plugins/ScheduledReports/config/tcpdf_config.php

```diff
@@ -60,6 +60,13 @@
                 $k_path_url = 'http://';
             }
             $k_path_url .= $_SERVER['SERVER_NAME'];
+            if (strpos($k_path_url, ':') === false
+                && !empty($_SERVER['SERVER_PORT'])
+                && $_SERVER['SERVER_PORT'] != 80
+                && $_SERVER['SERVER_PORT'] != 443
+            ) {
+                $k_path_url .= ':' . $_SERVER['SERVER_PORT'];
+            }
```

Comment:
Any reason for not directly using
```php
$k_path_url .= Url::getHostFromServerNameVar();
```

Reply:
I thought this was part of a library so I didn't want it to depend on Matomo, but looking at the path above, I guess it's actually Matomo code?

Reply:
yes, it's our configuration for tcpdf. Even though I think some parts of that config could maybe meanwhile be removed.